src/server/user-controller.js

```diff
@@ -222,31 +220,20 @@ class UserController {
     res.status(201).send({ user: updatedUser });
   }
 
-  async handlePostUserLogin(req, res, next) {
-    const failedAttemptsCount = await this.requestLimitRecordService.getCount({ req });
-    if (failedAttemptsCount >= FAILED_LOGIN_ATTEMPTS_LIMIT) {
-      throw new TooManyRequests();
-    }
-
-    passport.authenticate('local', async (err, user) => {
+  handlePostUserLogin(req, res, next) {
```

Comment:
This function is basically reset to the state before https://educandu.atlassian.net/browse/EDU-976